Add Rule #108: Democratic Voting on New Rules Modifications

WhiteFurryPie opened the pull request:
**Rules** #108

This new rule adds makes all newfound rules to be reviewed and modified by all players EXCEPT the one which proposes for the rule.

McCsocks commented:
Agreed 2/2

stolksdorf commented:
I like it, but I'd like some clarification on process of transfer. Not really something we'd add to the rule, but something I would add to the how2play.md

> To start a transfer of any kind between two players, one of those players submits a pull request for the scoreboard.md outlining the exchange. The other participant must approve the proposed exchange in the comments. Once approved a Mod will merge the change in.

Sound good?

mimosacb commented:
Works for me!

Meggeroni commented:
Awesome! Agreed 3/3. What do you guys want for your MBs?

JMTyler commented:
Agreed 4/4

![Throwin' mad points](https://i.imgur.com/8bB7J7n.gif)

stolksdorf commented:
Agreed. 5/5

we have majority!

labria committed:
Add myself

WhiteFurryPie committed:
Adding Oleg

WhiteFurryPie committed:
Merge pull request #1 from labria/WhiteFurryPie-patch-1

Adding Oleg

WhiteFurryPie committed:
Update rules.md

Adds the rule which makes a democratic voting on modifying the newfound rules whenever one is to be elected.
